config_test.go

```diff
@@ -323,3 +337,57 @@ func TestFilterCommandDeprecated(t *testing.T) {
 		}
 	}
 }
+
+var ConfigIgnoreTests = []struct {
+	name         string
+	ignore       *ecspresso.ConfigIgnore
+	resourceTags []types.Tag
+	expectedTags []types.Tag
+}{
+	{
```

Comment:
[nits] I thought it would be better to add test case to check case-sensitive.

> Tag keys and values are case-sensitive.

https://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_CreateService.html#ECS-CreateService-request-tags